MAUI/Circular-Charts/RadialBarChart.md

```diff
@@ -0,0 +1,433 @@
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart type in MAUI Chart](Chart-Types_images/maui_radialbar_chart.png)
+
+## Changing the radial bar size
+
+You can use the [Radius]() property to change the radial bar chart size. The default value of the radius is [0.8]().
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        Radius = "0.5"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.Radius = 0.5;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart radius size in MAUI Chart](Chart-Types_images/maui_radius.png)
+
+## Changing the radial bar inner radius
+
+The [InnerRadius]() property of radial bar series is used to define the inner circle. The default value of this property is `0.4`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        InnerRadius = "0.2"/>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.InnerRadius = 0.2;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart inner radius customization in MAUI Chart](Chart-Types_images/maui_inner_radius.png)
+
+## CapStyle customization
+
+The [CapStyle]() property of the radial bar series is used to specify the shape of the start and end points of the circular segment. The default value of this property is `Both.Flat`.
+
+The following types are available for [CapStyle]() property.
+
+ * [BothFlat]() - Indicates that a flat shape should appear at the start and end positions of the segment.
+
+ * [BothCurve]() - Indicates that a curve shape should appear at the start and end positions of the segment. 
+
+ * [StartCurve]() - Indicates that a curve shape should appear at the start position of the segment.
+
+ * [EndCurve]() - Indicates that a curve shape should appear at the end position of the segment.
+
+### BothFlat
```

Comment:
Remove all sub headings and use single code snippet and image for both curve alone.

Reply:
I have removed all the sub heading in cap style customization  section